docs/NativeCodeGeneration.md

```diff
@@ -73,13 +77,13 @@ driver skeleton.
 
 All Windows Drivers need to export a static function labelled &quot;DriverEntry&quot; that is invoked by the OS
 (Operating System) during loading of the driver. In addition to the standard driver related functionality, the driver
-entry skeleton would also register three [Network Module Registrar (NMR)](https://docs.microsoft.com/en-us/windows-hardware/drivers/network/introduction-to-the-network-module-registrar)
+entry skeleton also registers three [Network Module Registrar (NMR)](https://docs.microsoft.com/en-us/windows-hardware/drivers/network/introduction-to-the-network-module-registrar)
```

Comment:
The driver actually provides address of metadata table as part of client data when attach happens. Will that be 1 contract?

Reply:
Updated.